Source/Fuse.Elements/Drawing/ImageFill.uno

```diff
@@ -110,7 +170,8 @@ namespace Fuse.Drawing
 			public float2 Origin, Size;
 			public float4 UVClip;
 			public Texture2D Texture;
-			public float2 TexCoordBias1, TexCoordBias2, TexCoordScale1, TexCoordScale2;
+			public float2 TexCoordBias1, TexCoordBias2, TexCoordScale1, TexCoordScale2, TexCoordOffset, ScaleOrigin;
+			public float ScaleFactor;
```

Comment:
Well, without doing that, everyone who uses ImageFill has to pay for the cost of potentially being able to scale. I don't think that's very reasonable.